### PR TITLE
Gixing websocket subscriptions for signatures and slot

### PR DESCRIPTION
**Pull request by godmodegalactus**
Updating signatures for few methods,
Fixing websockets signatures.
Removing unwanted tests

### CODE REVIEW

File: quic-forward-proxy-integration-test/Cargo.toml [hunk 1]
```diff
@@ -18,7 +18,7 @@ solana-sdk = { workspace = true }
 solana-streamer = { workspace = true }
 solana-transaction-status = { workspace = true }
 solana-net-utils = { workspace = true }
-rustls = { workspace = true, features = ["dangerous_configuration"]}
```
Comment (grooviegermanikus) [inline quic-forward-proxy-integration-test/Cargo.toml:21]: why did you remove that?

File: tests/client.test.ts [hunk 2]
```diff
@@ -24,19 +24,61 @@ function createTransaction(): Transaction {
     return transaction;
 }
 
+// test('send and confrim with websockets', async () => {
```
Comment (grooviegermanikus) [inline tests/client.test.ts:27]: what is that commented code?

File: tests/client.test.ts [hunk 3]
```diff
@@ -46,45 +88,45 @@ test('send and confirm transaction', async () => {
...
-});
-
-
-test('get leader schedule', async () => {
-    {
-        const leaderSchedule = await connection.getLeaderSchedule();
-        expect(Object.keys(leaderSchedule).length > 0);        
-    }
-});
+// test('get epoch info', async () => {
```
Comment (grooviegermanikus) [inline tests/client.test.ts:91]: commented code - decide what to do with it

**Review by grooviegermanikus — DISMISSED** [review]

File: quic-forward-proxy-integration-test/Cargo.toml (hunk 1 above)
Comment (godmodegalactus) [reply quic-forward-proxy-integration-test/Cargo.toml:21]: not needed.

File: tests/client.test.ts (hunk 3 above)
Comment (godmodegalactus) [reply tests/client.test.ts:91]: Not needed really

File: tests/client.test.ts (hunk 2 above)
Comment (godmodegalactus) [reply tests/client.test.ts:27]: will remove it
